container-host-files/etc/hcf/config/role-manifest.yml

```diff
@@ -278,49 +278,6 @@ roles:
       properties.cf_mysql.mysql.galera_healthcheck.endpoint_password: ((MYSQL_GALERA_HEALTHCHECK_ENDPOINT_PASSWORD))
       properties.cf_mysql.proxy.api_password: ((MYSQL_PROXY_ADMIN_PASSWORD))
       properties.cf_mysql.proxy.proxy_ips: '["mysql-proxy.((KUBE_SERVICE_DOMAIN_SUFFIX))"]'
-- name: mysql-broker
```

Comment:
Wait what? That doesn't seem right. Why are we deleting it (even though it's terrible, it's a thing).

Reply:
This should have been removed a couple of sprints ago, it was only meant as a demo talk point while USB wasn't available.

Reply:
Then we should do that in a different PR, not this one. This would make it difficult to figure out later :)